SignalServiceKit/src/Util/OWSAnalyticsEvents.h

```diff
@@ -0,0 +1,139 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface OWSAnalyticsEvents : NSObject
+
+- (instancetype)init NS_UNAVAILABLE;
+
+#pragma mark - Code Generation Marker
```

Comment:
Maybe a comment to tie this back to the python script to talk about how this code was generated.